library/vsphere_vm

```diff
@@ -6,7 +6,26 @@ from pyVmomi import vim, vmodl
 from pyVim import connect
 
 PYTHON_IS_MODERN = sys.version_info >= (2, 7, 9)
-
+powerStates = {
```

Comment:
Use constants for these in underscore uppercase (e.g. POWER_STATES) and single quotes everywhere